Updates package instructions

svalo opened the pull request:
On ubuntu 16.04 the proposed instructions don't work. In order to download the index of packages https transport is needed and repos should be accessed over https otherwise the following happens
```
Ign:1 http://nightly.odoo.com/11.0/nightly/deb ./ InRelease
Err:2 http://nightly.odoo.com/11.0/nightly/deb ./ Release
  403  Forbidden

```

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

mart-e commented:
Hello,

Thank you for the CLA signature but it seems there is still an issue.

- you are signing _using_ the address `svalo@users.noreply.github.com`
- you are signing _for_ the address `svalo@libersoft.it`

The email address in the CLA must be the same as for the signature for the verification to succeed.
You can either change the your [git configuration](https://help.github.com/articles/setting-your-email-in-git/) or modify the email in the CLA signature.

Thanks

svalo commented:
@mart-e Excuse me, as it was a quick commit I did it on github without noticing which email it uses as default in the commit. It should be correct now. Thanks for the feedback

d-fence commented:
@svalo It's very strange that you got a '403 Forbidden'. AFAIK ubuntu does not enforce 'https', even http://archive.ubuntu.com/ is not served over https. The 403 error code came from our servers and it should not. Do you have a proxy server in between ?
As our package is signed, it does not add more security to download it over https. If an attacker want to modify the package during its download, the signature will be invalid and package will not be installed.
Also, we would like to have the installation procedure as simple as possible. That's why we want to avoid the install third party package (here apt-transport-https).
Finally, I tested the procedure against a fresh install of Ubuntu Xenial and it works over http.

svalo commented:
@d-fence 
> The 403 error code came from our servers and it should not. Do you have a proxy server in between ?

I checked the server again, I do have a proxy server in between, as I added other repos on other machines using same proxy I excluded that was an issue with the proxy. Today I doublechecked the proxy conf and noticed that other repositories were in whitelist while the one from odoo was not. 
I updated the proxy conf and I no longer receive the 403. Apt was configured to use the proxy only on http and not https. This explain why https was working. Excuse me for the false alarm, I'm closing this PR
